src/models/phoneNumber.model.ts

```diff
@@ -1,7 +1,8 @@
 import { model, Schema, Model } from 'mongoose';
+import { ICountry } from './country.model';
 
 export interface IPhoneNumber extends Document {
-  country_id: string;
+  country_id: ICountry['_id'];
```

Comment:
Giờ em mới biết có kiểu như này nữa à chị 
Vì cái interface ICountry đâu có trường này ta [_id]

Reply:
Do interface `ICountry extends Document` nên có mặc định có thuộc tính `'_id'`

Em tham khảo bài viết này nha:
https://tomanagle.medium.com/strongly-typed-models-with-mongoose-and-typescript-7bc2f7197722